### DISCUSSION TITLE
[Solved] Cannot install MPD on Windows.

### DISCUSSION THREAD

**Discussion by Francy2007** [MusicPlayerDaemon/MPD#1615] [Q&A]
Hello everyone,
I have Windows 10 Pro and I can't install MPD with mpd.exe (the installation procedure doesn't go on). How can I solve the problem?
Sincerely.

Vincenzo

**Comment by MaxKellermann** [comment]
What installation procedure do you mean?

**Reply by Francy2007** [reply]
As with all .exe files, I double-click with the left mouse button, but no program or service is installed. What am I doing wrong? Do I have to do anything else instead?
Sincerely.

Vincenzo

**Reply by MaxKellermann** [reply]
`mpd.exe` is the actual program, not an installer, and double-clicking it will start MPD. Copy it anywhere you want, there is no installer.

**Reply by Francy2007** [reply]
Good morning,
if I open mpd.exe with a double click of the mouse a window opens but only for a moment. Shouldn't it stay open with MPD running?
Sincerely.

Vincenzo

**Reply by Francy2007** [reply]
Good evening,
I ask Max Kellermann if he could kindly answer. Is it possible to solve the problem?
Sincerely.


Vincenzo

**Reply by Francy2007** [reply]
Hello everyone,
no answer ............ not even from Max Kellermann ....... At this point I think that mpd.exe is not working.
Having also Ubuntu, I also tried the compilation for Windows following the directions on www.musicpd.org, but I had no results, also because the explanation is very bad (maybe only a programmer can understand it, while it should be explained better and for everyone ). Obviously, on Windows, you will have to do without MPD .......
Sincerely.

Vincenzo.

**Reply by Francy2007** [reply]
Hello everyone,
I managed to get mpd.exe to work and if it can be useful I explain how I did it.
I created the "mpd" folder in C: \ and inside it I created the "playlists" folder.
Inside the "mpd" folder I inserted a compiled .conf file (with Notepad) as follows:

-----------------------------------------------------
music_directory "C: / Users / vince / Music"
playlist_directory "C: / mpd / playlists"
db_file "C: /mpd/mpd.db"
log_file "C: /mpd/mpd.log"
port "6600"
auto_update "yes"
input {
            plugin "local" 
}

audio_output {
    type "wasapi"
    name "Speakers"
    device "Speakers (AudioQuest DragonFly)"
    mixer_control "PCM"
    mixer_type "software"
    format "44100: 24: 2"
    enabled "yes"
    enumerate "no"
    exclusive "no"
    dop "no"
}

playlist_plugin {
       name "m3u"
       enabled "yes"
       as_directory "yes"
}
decoder {
             enabled "no"
             plugin "wildmidi" 
             config_file "/etc/timidity/timidity.cfg"
}

decoder {
             enabled "yes"
             plugin "ffmpeg" 
}
----------------------------------------------- 


I then created a .bat file compiled as follows:

---------------------------
C:
cd C: \ mpd
mpd mpd.conf
-----------------------------

Just double click the left mouse button to start MPD.

I am using Stylophone (Microsoft app) to use MPD.
My music files are local so on Stylophone the IP address indicated is 127.0.0.1 (it is the same for all if files are local).
Finally, i congratulate Max Kellermann on the great sound of MPD.
Sincerely.


Vincenzo
